cmd/openshift-tests/e2e.go

```diff
@@ -14,7 +14,46 @@ import (
 )
 
 func isDisabled(name string) bool {
-	return strings.Contains(name, "[Disabled")
+	if strings.Contains(name, "[Disabled") {
+		return true
+	}
+
+	return shouldSkipUntil(name)
+}
+
+// shouldSkipUntil allows a test to be skipped with a time limit.
+// the test should be annotated with the 'SkippedUntil' tag, as shown below.
+//  [sig-api-machinery] testing foo [SkippedUntil:05092022] [Suite:openshift]
+// The specified date should conform to the 'MMDDYYYY' format.
+// if the specified date in the tag has not passed yet, the test
+// will be skipped by the runner.
+func shouldSkipUntil(name string) bool {
+	_, after, found := cut(name, "[SkippedUntil:")
```

Comment:
How about using something like:
```
re, err := regexp.Compile(`\[SkippedUntil:(\d{6})\]`)
s := re.FindStringSubmatch(name)
```
this will return you 2-element array, where the first is the entire `[SkippedUntil:123456]` and the 2nd will be just `123456`.

Reply:
`SkippedUntil:<Date>,blocker-bz/<number>` to make the requirement clear.